### DISCUSSION TITLE
Avalonia UI !

### DISCUSSION THREAD

**Discussion by lukewire129** [JamesnetGroup/navigationbar#4] [Show and tell]
I did it exactly the same way

![KakaoTalk_20240118_144907899](https://github.com/vickyqu115/navigationbar/assets/54387261/fc11fcc0-7c2d-4d22-a0d7-2b9d4947765e)

https://github.com/lukewire129/AvaloniaNavigationBar

**Comment by vickyqu115** [comment]
Oh! That's really interesting. I thoroughly reviewed the source code on GitHub and it was quite fascinating to see how similar it is to the WPF structure I created!
I expect it will be of great help to many people who are getting started with AvaloniaUI!

**Reply by lukewire129** [reply]
Good Good~!

**Comment by furesoft** [comment]
Please implement that the item count don't has to be 5

**Reply by lukewire129** [reply]
Ok, 
We changed the items panel to StackPanel, so we processed it regardless of the number of items

Thank you!

**Reply by furesoft** [reply]
![image](https://github.com/vickyqu115/navigationbar/assets/4117602/32b52646-5bfc-4e59-8ed4-c689c048ee8b)

it seems that the selection circle and clip does not be moved correctly

**Reply by lukewire129** [reply]
ya..
I think it's because the border area on the left is a little wider, so I need to check it out.

**Reply by lukewire129** [reply]
Added the Width value for ListBoxItem.
But it also seems that the icons are tilted slightly to the right.

![image](https://github.com/vickyqu115/navigationbar/assets/54387261/9f64dd59-54ab-460d-aeb5-e0905736f29b)
![image](https://github.com/vickyqu115/navigationbar/assets/54387261/973a1584-fdc1-4696-a6ea-97c6181b1989)
![image](https://github.com/vickyqu115/navigationbar/assets/54387261/4f700a8f-697b-43ea-a039-238a955843ae)
![image](https://github.com/vickyqu115/navigationbar/assets/54387261/f284d98d-3693-4557-8935-9ad9f7f869d2)
![image](https://github.com/vickyqu115/navigationbar/assets/54387261/4de3fd43-db29-4dcc-89b4-81b788794de2)

**Reply by lukewire129** [reply]
Navigation Extentions!

https://github.com/vickyqu115/navigationbar/assets/54387261/92e81534-2f81-4489-8715-47f9c6207946
